### DISCUSSION TITLE
Unclear how to resolve a review

### DISCUSSION THREAD

**Discussion by idbrii** [community/community#9709] [Pull Requests]
I frequently have reviews that are stuck in the "1 review requesting changes" stage. As far as I can tell, I've resolved everything. And don't have any ideas how to proceed. I usually request another review and I think the maintainer pushes it through?

Take https://github.com/pfirsich/makelove/pull/14

It has one requested change:

![image](https://user-images.githubusercontent.com/43559/148419552-659bb302-d93c-47f9-87ae-c6082462f6fa.png)

I've clicked the "Resolve Conversation" button. But it still says there are changes requested:

![image](https://user-images.githubusercontent.com/43559/148419754-13fa579f-a0b5-449f-bbdf-4adc7d0d31c4.png)

If I click on "View changes" in the first screenshot, I get "We went looking everywhere, but couldn’t find those commits.".

If I click on "See review" it highlights the resolved conversation.

![image](https://user-images.githubusercontent.com/43559/148420006-aee41d32-7e0a-483d-a9c0-bcc5478aba67.png)

Possibly this is because I force-pushed *before* I resolved the conversation? But why wouldn't I wait until I fixed something before resolving? Or maybe I force-pushed *after* I resolving? That's also reasonable if I'm marking things as done as I finish them locally.

[The documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews) implies that I'm doing the right thing. I'm not sure if I'm encountering a bug or if the flow is simply unclear.

**Comment by marianatuma** [comment]
I'm having the same problem. Any ideas on how to solve this?

**Reply by idbrii** [reply]
I usually add a comment to ensure the other person received a notification. Probably the biggest improvement they could make to this flow is to give you feedback when someone was notified.
